File: PRAK104-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
a = 400000
b = 350000
diskonA = 13
diskonB = 21
hasilA = (a * diskonA) / 100
hasilB = (b * diskonB) / 100
totalA = round (a - hasilA)
totalB = round (b - hasilB)

print(f'Harga sepatu A adalah {a}')
print(f'Harga sepatu B adalah {b}')

print(f'Sepatu A mendapat diskon {diskonA}% sehingga harganya menjadi {totalA}')
print(f'Sepatu B mendapat diskon {diskonB}% sehingga harganya menjadi {totalB}')

File: MODUL 1/PRAK104-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

int main() {
    int a = 400000;
    int b = 350000;
    int diskonA = 13;
    int diskonB = 21;
    int hasilA = a * diskonA / 100;
    int hasilB = b * diskonB / 100;

    printf("Harga sepatu A bernilai %d\n", a);
    printf("Harga sepatu B bernilai %d\n", b);
    
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", a - hasilA);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d\n", b - hasilB);

    return 0;
}
